rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -61,21 +61,22 @@ ppu_static_module::ppu_static_module(const char* name)
 	ppu_module_manager::register_module(this);
 }
 
-std::unordered_map<std::string, ppu_static_module*>& ppu_module_manager::access()
-{
-	static std::unordered_map<std::string, ppu_static_module*> map;
-
-	return map;
-}
+// We move the static unordered_map up into the ppu_module_manager definition
```

Comment:
What's the use of this commented code?

Reply:
I thought it was staying inline with the 'Comment every hack you do, every snippet you comment out and every improvable code.' (i.e. the every snippet you comment out)
If you'd prefer it just gone, I can do that.